docs/config.rst

```diff
@@ -418,10 +418,12 @@ Complete list of settings that you can put into ``testenv*`` sections:
 
     A list of wildcard environment variable names which
     shall be copied from the tox invocation environment to the test
-    environment when executing test commands.  If a specified environment
+    environment when executing test commands. If a specified environment
```

Comment:
I'd leave this alone, it's unrelated to your change

Reply:
Sure @asottile, I removed it